plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -77,6 +89,9 @@ private static boolean inCollection(String item, final Collection<String> texts)
 	 */
 	public static boolean selectorsOverlap(Map<String, String> source, Map<String, String> target) {
```

Comment:
still thinking that the logic is not "overlapping" but rather "containsAll" like one can see in the code. I'd change the method name for added expressivenes